### ISSUE TITLE
A11y Regression: Individual image lightbox toggle has no accessible name

### ISSUE THREAD

**Issue by joedolson** [WordPress/gutenberg#78413]
### Description

Reported by https://github.com/hagege; discussion at https://wordpress.slack.com/archives/C02RP4X03/p1779084285204059

In the 7.0 RC, individual images with the Lightbox effect applied have a button with no `aria-label`. This appears to come from the [initial commit to add gallery support](https://github.com/WordPress/gutenberg/commit/676fabe8cdbec7fbfd02926fabcd961d31a1bb1d#diff-1e4aa29ea183efda9c795271dc1acf6e8049355db104d0dca5b0ea5caf7f619cR251), which sets the individual `aria-label` to null.

The gallery lightbox is not impacted. This works on WP 6.9.4.

### Step-by-step reproduction instructions

1. Insert a single image.
2. Select the expand on click option from the link  menu.
3. Inspect the lightbox-toggle control on the front-end for an `aria-label`.

### Screenshots, screen recording, code snippet

<img width="1917" height="851" alt="Code showing missing aria-label" src="https://github.com/user-attachments/assets/7c4347fe-b398-4b21-afb8-add1c9184498" />

### Environment info

- Current WP RC, no plugins.

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [x] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure
